README.md

```diff
@@ -151,13 +123,15 @@ This following information is parsed from the integration:
 - JumpBox
   - JUMP_SVR_HOST
   - JUMP_SVR_USER
-  - JUMP_SVR_PRIVATE_KEY (if a random one is generated)
+  - JUMP_SVR_RWX_FILESTORE_PATH
 - Postgres (When V4_CFG_POSTGRES_TYPE is set to external)
   - V4_CFG_POSTGRES_ADMIN_LOGIN
   - V4_CFG_POSTGRES_PASSWORD
   - V4_CFG_POSTGRES_FQDN
+  - V4_CFG_POSTGRES_CONNECTION_NAME
+  - V4_CFG_POSTGRES_SERVICE_ACCOUNT
 
-### Customizations
+### Customize Deployment Overlays
 
 viya4-deployment fully manages the kustomization.yaml file. Users can make change by adding custom overlays into sub-folders under the site-config folder. If this is the first time you are running the tool and you need customizations, you can create the folder structure below.
```

Comment:
"Users can make changes by adding custom overlays into sub-folders under the site-config folder. If this is the first time  running the tool and customizations will be provided, create the folder structure below."

Reply:
updated